data/templates/Detector_KubernetesService.csx

```diff
@@ -15,9 +15,8 @@ public async static Task<Response> Run(DataProviders dp, OperationContext<AzureK
 {
     res.Dataset.Add(new DiagnosticData()
     {
-        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), "Aks", "AKSprod", null, "GetQuery"),
-        RenderingProperties = new Rendering(RenderingType.Table)
-		{
+        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt)),
```

Comment:
Please change the line to 
dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), null, "GetQuery")